endpoints/api/test/test_immutability_policy.py

```
@@ -0,0 +1,213 @@
+import pytest
+
+from data.model import immutability
+from endpoints.api.immutability_policy import (
+    OrgImmutabilityPolicies,
+    OrgImmutabilityPolicy,
+    RepositoryImmutabilityPolicies,
+    RepositoryImmutabilityPolicy,
+)
+from endpoints.api.test.shared import conduct_api_call
+from endpoints.test.shared import client_with_identity
+from test.fixtures import *  # noqa: F401, F403
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)